packages/metro-source-map/package.json

```diff
@@ -12,8 +12,9 @@
     "cleanup-release": "test ! -e build && mv src build && mv src.real src"
   },
   "dependencies": {
-    "@babel/traverse": "^7.20.0",
-    "@babel/types": "^7.20.0",
+    "@babel/traverse": "^7.25.3",
+    "@babel/traverse--for-generate-function-map": "npm:@babel/traverse@^7.25.3",
```

Comment:
Why was this dependency with referer created?

Reply:
Our security tools are highlighting this as a potential security issue.

Reply:
@jerone- hey! the PR summary including this information was cut. Thanks for noticing. I've updated the summary to include the reasoning behind installing `@babel/traverse` twice.

The tool flagging it as a potential security issue is concerned about "detected resolved URL for package with a different name" but this is totally expected, so I think you can ignore or mark this concern as resolved.

Reply:
Thanks for the update. I have whitelisted this change.

The tool that giving the warning was lockfile linting: https://github.com/lirantal/lockfile-lint/tree/main/packages/lockfile-lint-api#validators